modules/ROOT/pages/customer-connector-upgrade.adoc

```diff
@@ -0,0 +1,162 @@
+= Customer Guide for Upgrading and Testing Custom Connectors for Java 17
```

Comment:
I guess this "Customer Guide" is necessary to distinguish from the Partner Guide? And the audience is the most important bit (so appears first)?

In general, we avoid self-referential terms like "Guide". If not required, then suggest something like:
Upgrading and Testing Custom Connectors for Java 17 (for Customers)

Question: who is the customer in this case? Is it our customer or our partner's customer? Ignore if this is obvious in context.